index.js

```diff
@@ -210,10 +210,13 @@ var lastResorts = [
   ['BEER', 'Beerware'],
   ['BOOST', 'BSL-1.0'],
   ['BSD', 'BSD-2-Clause'],
+  ['CDDL', 'CDDL-1.1'],
```

Comment:
I'd love to see that blog post :-)
At @snyk, where I work, we use this but I can't say more since it's still in the works.
A new version with this will be great. Can't say it's urgent, though, since I have a wrapper around this library exactly for such cases. I just think the reasonable thing is to donate back to the project when I find stuff that I believe might benefit others, hence this pr.